rio/srcrobot/subsystems/Shooter.py

```diff
@@ -65,6 +65,9 @@ def setShooterVelocity(self, velocity):
 
         self.currentShotVelocity = Conversions.MPSToRPS(velocity,  self.wheelCircumference)*self.gearRatio
 
+    def shootVelocity(self, velocity):
```

Comment:
adding -> Command to the end of the def would be nice to document the return type.